### DISCUSSION TITLE
Ideas around frames :-)

### DISCUSSION THREAD

**Discussion by Striker62** [photobooth-app/photobooth-app#197] [Ideas]
First of all perfect work. I will build a photobooth for my sons weeding
He Loves Asterix and she loves Harry Potter. I would like to setup two frames related to both themes.
It would be great if they can select the frame, at the liveview, they would like have .
Second would be the great if the frame can be shown in the liveview, to have the chance to be in the right possition  .
(sorry for my bad english.

**Comment by mgineer85** [comment]
Right now none of the functions is implemented unfortunately.
It could be possible to add later in version 2 once finished. 
Feel free to open a feature request to describe the expected workflow.

**Comment by Striker62** [comment]
will do. I'm new in setting up such a request. do you have a template or a description what you expact

**Comment by mgineer85** [comment]
Yes, just go over to the issues tab and there choose feature request. Fill what is possible, skip else. There are no fixed rules 😀
